lib/tools/system-calls.js

```diff
@@ -608,4 +614,21 @@ systemCallMethods.ls = async function (remotePath) {
   }
 };
 
+systemCallMethods.fileSize = async function (remotePath) {
+  try {
+    let files = await this.ls(remotePath, ['-la']);
```

Comment:
I guess we could? But this makes less sense to me than a regular expression, and also `exec` doesn't handle pipes very well.